frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/widgets/mobile_field_name_text_field.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:appflowy/plugins/database_view/application/field/field_editor_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class MobileFieldNameTextField extends StatefulWidget {
+  const MobileFieldNameTextField({
+    this.text,
+    super.key,
+    this.textEditingController,
+  });
+
+  final String? text;
+  final TextEditingController? textEditingController;
+
+  @override
+  State<MobileFieldNameTextField> createState() =>
+      _MobileFieldNameTextFieldState();
+}
+
+class _MobileFieldNameTextFieldState extends State<MobileFieldNameTextField> {
+  late TextEditingController controller;
+  @override
+  void initState() {
+    super.initState();
+    controller = widget.textEditingController ?? TextEditingController();
+    if (widget.text != null) {
+      controller.text = widget.text!;
+    }
+  }
+
+  @override
+  void dispose() {
+    controller.dispose();
```

Comment:
If the controller is not created internally, then you shouldn't dispose it. The entity that creates it should be responsible for disposing it.